.github/workflows/merge-change-log-to-main.yml

```diff
@@ -0,0 +1,44 @@
+name: Merge change log to main
+on:
+  workflow_dispatch:
+
+jobs:
+  create-pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          if [[ $GITHUB_REF_NAME != release/* ]]; then
+            echo this workflow should only be run against release branches
+            exit 1
+          fi
+
+      - uses: actions/checkout@v3
+        with:
+          # this workflow is run against the release branch (see usage of GITHUB_REF_NAME below)
+          # but it is creating a pull request against main
+          ref: main
+          # history is needed to run format-patch below
+          fetch-depth: 0
+
+      - name: Set git user
+        run: .github/scripts/set-git-user.sh
+
+        # this will fail if there have been conflicting change log updates introduced in main
```

Comment:
When does this happen? We make PR author to update the CHANGELOG entries.

Reply:
good point, this is probably going to fail a lot and we should think of another option.

I guess now that the date is getting stamped onto the change log during prepare-release-branch, this step should only be needed for patch releases.

I'll change this to just be a manual step of merging the patch release change log back into main unless anyone has a better idea.